Cargo.toml

```diff
@@ -23,11 +23,15 @@ subtle = { version = "2.1.1", default-features = false }
 digest = { version = "0.10.5", default-features = false, features = ["alloc", "oid"] }
 pkcs1 = { version = "0.7.1", default-features = false, features = ["alloc", "pkcs8"] }
 pkcs8 = { version = "0.10", default-features = false, features = ["alloc"] }
-serde = { version = "1.0.103", optional = true, default-features = false, features = ["derive"] }
-sha2 = { version = "0.10.6", optional = true, default-features = false, features = ["oid"] }
 signature = { version = "2", default-features = false , features = ["digest", "rand_core"] }
 zeroize = { version = "1", features = ["alloc"] }
 
+# optional dependencies
+const-oid = { version = "0.9", optional = true }
```

Comment:
This is already a transitive dependency of `pkcs1` and `pkcs8`, which are both hard dependencies. You might as well make it a hard dependency.

Reply:
done